pkgs/development/python-modules/aioasuswrt/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, asyncssh
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, pytest-asyncio
+, pytest-cov
+, pytest-mock
+, pytestCheckHook
+, pythonAtLeast
+}:
+
+buildPythonPackage rec {
+  pname = "aioasuswrt";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "kennedyshead";
+    repo = pname;
+    rev = "V${version}";
+    sha256 = "1vqihb5fgphb03867b90aayfnnrssda7vs97f84q7s4cwgjxi9lg";
+  };
+
+  patches = [
+    (fetchpatch {
+      # Remove pytest-runner, https://github.com/kennedyshead/aioasuswrt/pull/63
+      url = "https://github.com/kennedyshead/aioasuswrt/pull/63/commits/e7923927648d5d8daccac1716db86db2a45fcb34.patch";
+      sha256 = "09xzs3hjr3133li6b7lr58n090r00kaxi9hx1fms2zn0ai4xwp9d";
+    })
+  ];
+
+  propagatedBuildInputs = [ asyncssh ];
+
+  checkInputs = [
+    pytest-asyncio
+    pytest-cov
+    pytest-mock
+    pytestCheckHook
+  ];
+
+  # Support for later Python releases, https://github.com/kennedyshead/aioasuswrt/issues/49
+  doCheck = pythonAtLeast "3.8";
```

Comment:
Should we disable the package on earlier versions entirely?

Reply:
Support for Python > 3.8 was merged. Waiting for a new release...

Reply:
Yes, release is available.